Gemfile

```diff
@@ -3,6 +3,11 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "3.0.1"
 
+gem "sass-rails"
+
+gem 'bootstrap-sass', '~> 3.4.1'
+
+gem 'jquery-rails'
```

Comment:
I think they are going backwards, because instead of using gems needed for Rails 3, they could take advantage of Rails 7. One gem that is being implemented to give styles new functionality is [cssbundling-rails](https://rubygems.org/gems/cssbundling-rails)

Reply:
You're right Yil, very good observation and thanks for sharing the link, thanks!